ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,6 +106,13 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+    	gocdbServiceType = "FTS"
+    	try:
+    		elementName	= getFTSServers("FTS3")[ 'Value' ][0]
+    	except:
+    		return S_ERROR( 'No FTS3 server specified in dirac.cfg (see Resources/FTSEndpoints)' )
```

Comment:
Well we have several FTS3 servers (one at CERN, one at RAL). And we want to know independently which one is in downtime or available. Is it clearer ?